### ISSUE TITLE
Changing Dropdown fields to 'allow multiple' and then changing it back to default causes runtime errors

### ISSUE THREAD

**Issue by am0** [verbb/formie#480]
**Description**

Getting this when switching a dropdown menu from 'Allow multiple' back to default. Had to remove the dropdown field entirely and re-add it, after which it worked.

![Screenshot 2021-06-01 at 11 23 55](https://user-images.githubusercontent.com/618467/120349065-ad960c00-c2f5-11eb-8573-e04349ed2ac4.png)

Thanks!

**Steps to reproduce**
1. Create a form and add a dropdown field with a few items
2. Check 'Allow multiple' for this field, and save
3. Add to front-end (in my case, via `{% set form = block.contactFormie.one() %}` and `{{craft.formie.renderForm(form)}}`)
4. Form displays as normal
5. Uncheck the dropdown field's 'Allow multiple' option and save
6. Will give this error message

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: Latest
- Craft version: Latest
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
